conf/base_config.yaml

```diff
@@ -107,12 +107,15 @@ encoder:
 
 generate:
     splits: [train, valid, test]
+    n_processes: 1
+    n_workers_per_process: 0
     n_batches: ${simulator.n_batches}
     batch_size: ${simulator.prior.batch_size}
     valid_n_batches: ${simulator.valid_n_batches}
     test_n_batches: ${simulator.valid_n_batches}
     max_images_per_file: 5000
     cached_data_path: ${paths.data}/cached_dataset
+    train_file_prefix: dataset
```

Comment:
You'll need to change the loading in CachedSimulatedDataset to use this too, since it's hardcoded as "dataset_" right now

Reply:
Done

Reply:
Also need to update the filenames for the validation and test datasets: 
https://github.com/prob-ml/bliss/blob/201ca1f7594c140490b19b863e6cd85e7f5b4788/bliss/simulator/simulated_dataset.py#L132

Reply:
Revamped design / use case of data generation/training. ptal at new commit.